plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,382 @@
+package whatap
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+
+	//"github.com/influxdata/telegraf/internal"
```

Comment:
Please remove this if not needed.

Reply:
Edited and applied. We ask for confirmation.